src/components/BookList.js

```diff
@@ -0,0 +1,43 @@
+export default function BookList() {
+  const booklist = [
+    {
+      title: 'The Hunger Games',
+      Author: 'Suzanne Collins',
+    },
+    {
+      title: 'Dune',
+      Author: 'Frank Herbert',
+    },
+    {
+      title: 'Capital in the Twenty-First Century',
+      Author: 'Suzanne Collins',
+    },
+  ];
+
+  return (
+    <div className="main-container">
+      <div className="bookList-container">
+        <div className="book-item">
+          {booklist.map((item) => (
+            <li key={item.id}>
+              {item.title}
+              <br />
+              {item.Author}
+              <br />
+              <br />
+            </li>
+          ))}
+        </div>
+        <hr />
+      </div>
```

Comment:
Good job creating a BookList component. According to the project requirements, you are expected to structure your components in the order below:

- [ ] Create a book component that you import in this BookList component, loop through the books array, and pass every single book into the single Book component as a prop, in which you will then receive the prop data inside the Book component (title, author) and render them in that component. Kindly refactor your code this way. This makes the BookList component lean, maintainable, and readable. 👍🏽
- [ ] In this single Book.js component, after rendering the props data(title, and author), there would be a button to delete or remove a book, Kindly add that 👍🏽